data/macos.yaml

```diff
@@ -369,7 +401,7 @@ name: MacOSMailAccounts
 doc: Mail Accounts
 sources:
 - type: FILE
-  attributes: {paths: ['%%users.homedir%%/Library/Mail/V2/MailData/Accounts.plist']}
+  attributes: {paths: ['%%users.homedir%%/Library/Mail/V[0-9]/MailData/Accounts.plist']}
```

Comment:
Have we seen all 0-9 versions? or is this 1-3 ?

Reply:
I have seen V2, V3 and V5, I thought 0-9 was good to be on the safe side.

Reply:
ack, SG; maybe add this to the doc string that V2, V3 and V5 have been observed?

Reply:
Done